src/components/Home.js

```diff
@@ -1,28 +1,23 @@
 import React, { PureComponent } from 'react'
-import { StyleSheet, Text, Button, View, Image } from 'react-native'
+import { StyleSheet, Button, View } from 'react-native'
 
-import mainStyles from '../styles'
-import Logo from '../ui/Logo'
 
 export default class Home extends PureComponent {
-    render() {
-        return (
-            <View style={mainStyles.container}>
-                <Logo />
-                <View style={styles.header}>
-                    <Button title="Transactions" />
-                    <Button title="Pay" />
-                    <Button title="Buy / Sell" />
-                </View>
-            </View>
-        )
-    }
+  render() {
+    return (
+      <View style={styles.header}>
+          <Button title="Transactions" />
+          <Button title="Pay" />
```

Comment:
@vryazanov what is diff between pay and buy/sell?

Reply:
@vladarefiev buy/sell is about actions with Zold exchange platform, BTC -> ZLD / ZLD -> BTC.
Pay is about transfer between wallets inside Zold